platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/iss/internal/ConsensusHashFinder.java

```diff
@@ -238,9 +238,6 @@ public void writePartitionData(final StringBuilder sb) {
                 .append("%) of total weight have reported a hash for round ")
                 .append(round)
                 .append(".\n");
-        if (consensusHash != null) {
-            sb.append("Consensus hash: ").append(consensusHash.toMnemonic()).append("\n");
```

Comment:
@cody-littley I removed this, since each partition reports its hash below. Just checking that you don't see any knock-on effects I'm not considering

Reply:
This actually is helpful I think, even if there is a little redundancy. The consensus hash is defined to be "the correct hash".

To derive the consensus hash, look at signature transactions in the order that they reach consensus. Choose the first partition that gathers >1/2 consensus weight. A catastrophic ISS is defined as a situation where there exists no consensus hash.

Here's an idea. You currently write the following for each partition.

```
            sb.append("  partition hash (unabbreviated): ")
                    .append(partition.getHash())
                    .append("\n");
```

What if you change it to this for the partition with the consensus hash?

```
            sb.append("  partition hash (consensus): ")
                    .append(partition.getHash())
                    .append("\n");
```

Reply:
Added back, with both mnemonic and standard hash